presto-hive/src/main/java/com/facebook/presto/hive/metastore/thrift/ThriftMetastoreUtil.java

```diff
@@ -586,4 +608,32 @@ private static OptionalLong toMetastoreDistinctValuesCount(OptionalLong distinct
         }
         return distinctValuesCount;
     }
+
+    public static Set<ColumnStatisticType> getSupportedColumnStatistics(Type type)
+    {
+        if (type.equals(BOOLEAN)) {
+            return ImmutableSet.of(NUMBER_OF_NON_NULL_VALUES, NUMBER_OF_TRUE_VALUES);
+        }
+        if (isNumericType(type) || type.equals(DATE) || type.equals(TIMESTAMP)) {
+            // TODO #7122 support non-legacy TIMESTAMP
+            return ImmutableSet.of(NUMBER_OF_NON_NULL_VALUES, MIN, MAX, NUMBER_OF_DISTINCT_VALUES);
```

Comment:
Nit: put min/max at the start or end, so that the `NUMBER_*` stats are together

Reply:
I'm wondering if `MAX` should be `MAX_VALUE`. That might be more consistent with `MAX_VALUE_SIZE_IN_BYTES`.

Reply:
> I'm wondering if MAX should be MAX_VALUE. That might be more consistent with MAX_VALUE_SIZE_IN_BYTES.

Yeah. That it will be less chance that it clashes on static import. Let me change that.